broker/adminapi.go

```diff
@@ -0,0 +1,46 @@
+package broker
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/frodenas/brokerapi"
+	"github.com/frodenas/brokerapi/auth"
+	"github.com/pivotal-golang/lager"
+)
+
+func NewAdminAPI(serviceBroker *Broker, logger lager.Logger, brokerCredentials brokerapi.BrokerCredentials) http.Handler {
+	router := newHTTPRouter()
+
+	router.Get("/admin/cells", cells(serviceBroker, router, logger))
+	router.Get("/admin/hello/{person}", hello(serviceBroker, router, logger))
```

Comment:
Was keeping it for the memories :) And its an example of parameters. But yeah, it can be removed.